hardware/raspberrypi/usb/README.md

```diff
@@ -12,48 +12,46 @@
 <a name="overview"></a>
 ## Overview for devices prior to Pi 4
 
-The number and type of USB ports on Raspberry Pi depends on the model. The Raspberry Pi Model B is equipped with two USB 2.0 ports; the B+, 2B, 3B and 3B+ have four USB 2.0 ports. The Pi 4 has two USB 2.0 ports and two USB 3.0 ports. In all models prior to the Pi 4, the USB ports connect to a combo hub/Ethernet chip, which is itself a USB device connected to the single upstream USB port on BCM2835. On the Pi 4, the USB hub chip is connected to the SoC using a PCIe bus.
+The number and type of USB ports on Raspberry Pi depends on the model. The original Raspberry Pi Model B was equipped with two USB 2.0 ports; the B+, 2B, 3B and 3B+ have four USB 2.0 ports. The Pi 4 has two USB 2.0 ports and two USB 3.0 ports. In all models prior to the Pi 4, the USB ports connect to a combo hub/Ethernet chip, which is itself a USB device connected to the single upstream USB port on the SoC. On the Pi 4, the USB hub chip is connected to the SoC using a PCIe bus.
```

Comment:
In contrast to other PRs, I wonder if it's worth being more specific here and actually detailing "Pi 4B" and "Pi 400" separately, as they have different numbers of USB ports?

Reply:
Thinking again about this, I think it would actually be better to delete this paragraph - folk will have already got this information from the product pages or will have a Pi in front of them, and the documentation section of the website is not where you would naturally go to find out this information.